libmscore/chordrest.cpp

```diff
@@ -1227,12 +1227,90 @@ void ChordRest::processSiblings(std::function<void(Element*)> func)
       }
 
 //---------------------------------------------------------
+//   nextArticulationOrLyric
+//---------------------------------------------------------
+
+Element* ChordRest::nextArticulationOrLyric(Element* e)
```

Comment:
fwiw, I might have taken this as an opportunity to consider betters names all around.  What we call "nextElement" is really mostly "nextChildElement", right?  Same with this.  "next-element" makes sense as a *command* name because user doesn't care if it is a child, sibling, parent, or what, but it seems ambiguous as a function name.  Whereas "nextArticulationOrLyric" seems overly specific.  Oh well, not really worth changing, as far as I am concerned.